snooty.toml

```diff
@@ -17,9 +17,9 @@ sharedinclude_root = "https://raw.githubusercontent.com/10gen/docs-shared/main/"
 driver = "java"
 driver-long = "MongoDB Java Driver"
 version = "4.7"
-full-version = "{+version+}.0-beta0"
+full-version = "{+version+}.0"
 
 package-name-org = "mongodb-org"
 api = "https://mongodb.github.io/mongo-java-driver/{+version+}"
 stable-api = "Stable API"
-mongocrypt-version="1.4.0" # ensure compatibilty with driver version
+mongocrypt-version = "1.4.0"                                    # ensure compatibilty with driver version
```

Comment:
Question: is the whitespace between the setting value and the comment necessary?

Reply:
I don't believe it is, that's just how my toml formatter saved it.